[2023-06-26T12:11:34.250+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:11:32.482465+00:00/task_id=load_seed_todl_once permission to 509
[2023-06-26T12:11:34.251+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:11:32.482465+00:00/task_id=load_seed_todl_once permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:11:32.482465+00:00/task_id=load_seed_todl_once'
[2023-06-26T12:11:34.296+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:11:32.482465+00:00/task_id=load_seed_todl_once permission to 509
[2023-06-26T12:11:34.298+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:11:32.482465+00:00/task_id=load_seed_todl_once permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:11:32.482465+00:00/task_id=load_seed_todl_once'
[2023-06-26T12:11:34.559+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5.load_seed_todl_once manual__2023-06-26T12:11:32.482465+00:00 [queued]>
[2023-06-26T12:11:34.575+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5.load_seed_todl_once manual__2023-06-26T12:11:32.482465+00:00 [queued]>
[2023-06-26T12:11:34.576+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-26T12:11:34.600+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): load_seed_todl_once> on 2023-06-26 12:11:32.482465+00:00
[2023-06-26T12:11:34.609+0000] {standard_task_runner.py:57} INFO - Started process 1324 to run task
[2023-06-26T12:11:34.615+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5', 'load_seed_todl_once', 'manual__2023-06-26T12:11:32.482465+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpikox7ja4']
[2023-06-26T12:11:34.621+0000] {standard_task_runner.py:85} INFO - Job 128: Subtask load_seed_todl_once
[2023-06-26T12:11:34.699+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:11:32.482465+00:00/task_id=load_seed_todl_once permission to 509
[2023-06-26T12:11:34.700+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:11:32.482465+00:00/task_id=load_seed_todl_once permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:11:32.482465+00:00/task_id=load_seed_todl_once'
[2023-06-26T12:11:34.705+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5.load_seed_todl_once manual__2023-06-26T12:11:32.482465+00:00 [running]> on host 84990cf1ddee
[2023-06-26T12:11:34.895+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5' AIRFLOW_CTX_TASK_ID='load_seed_todl_once' AIRFLOW_CTX_EXECUTION_DATE='2023-06-26T12:11:32.482465+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-26T12:11:32.482465+00:00'
[2023-06-26T12:11:34.898+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-26T12:11:34.899+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt &&             dbt seed --profiles-dir .']
[2023-06-26T12:11:34.913+0000] {subprocess.py:86} INFO - Output:
[2023-06-26T12:11:39.390+0000] {subprocess.py:93} INFO - [0m12:11:39  Running with dbt=1.5.1
[2023-06-26T12:11:44.492+0000] {subprocess.py:93} INFO - [0m12:11:44  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-26T12:11:44.493+0000] {subprocess.py:93} INFO - There are 2 unused configuration paths:
[2023-06-26T12:11:44.494+0000] {subprocess.py:93} INFO - - models.badge_5.time_of_day
[2023-06-26T12:11:44.495+0000] {subprocess.py:93} INFO - - models.badge_5.create_stage
[2023-06-26T12:11:44.567+0000] {subprocess.py:93} INFO - [0m12:11:44  Found 1 model, 0 tests, 0 snapshots, 0 analyses, 455 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-26T12:11:44.574+0000] {subprocess.py:93} INFO - [0m12:11:44
[2023-06-26T12:11:47.164+0000] {subprocess.py:93} INFO - [0m12:11:47  Concurrency: 200 threads (target='dev')
[2023-06-26T12:11:47.166+0000] {subprocess.py:93} INFO - [0m12:11:47
[2023-06-26T12:11:47.258+0000] {subprocess.py:93} INFO - [0m12:11:47  1 of 1 START seed file ***.time_of_day_lu .................................. [RUN]
[2023-06-26T12:11:50.631+0000] {subprocess.py:93} INFO - [0m12:11:50  1 of 1 OK loaded seed file ***.time_of_day_lu .............................. [[32mINSERT 24[0m in 3.37s]
[2023-06-26T12:11:50.706+0000] {subprocess.py:93} INFO - [0m12:11:50
[2023-06-26T12:11:50.708+0000] {subprocess.py:93} INFO - [0m12:11:50  Finished running 1 seed in 0 hours 0 minutes and 6.13 seconds (6.13s).
[2023-06-26T12:11:50.765+0000] {subprocess.py:93} INFO - [0m12:11:50
[2023-06-26T12:11:50.767+0000] {subprocess.py:93} INFO - [0m12:11:50  [32mCompleted successfully[0m
[2023-06-26T12:11:50.769+0000] {subprocess.py:93} INFO - [0m12:11:50
[2023-06-26T12:11:50.771+0000] {subprocess.py:93} INFO - [0m12:11:50  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2023-06-26T12:11:51.633+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-26T12:11:51.673+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5, task_id=load_seed_todl_once, execution_date=20230626T121132, start_date=20230626T121134, end_date=20230626T121151
[2023-06-26T12:11:51.748+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-26T12:11:51.792+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
